lit/architecture/architecture.lit

```diff
@@ -4,7 +4,7 @@
 
 Concourse is a fairly simple distributed system built up from the following
 components. You'll see them referenced here and there throughout the
-documentation, so you may want to skim this page just to get an idea of what
+documentation, so you may want to skip this page just to get an idea of what
```

Comment:
The original (skim) is actually correct. Skip basically means to ignore / bypass where as skim means to read quickly. If the meaning of skim isn't clear, it could be changed to `...may want to quickly review...` instead.

Reply:
Oh, Now I understood the intend. 
It's come from my poor language skill. thank you for explain it.